_data/simple-icons.json

```diff
@@ -4941,7 +4944,7 @@
         {
             "title": "Kotlin",
             "hex": "0095D5",
-            "source": "https://resources.jetbrains.com/storage/products/kotlin/docs/kotlin_logos.zip",
+            "source": "https://kotlinlang.org/docs/faq.html#where-can-i-get-an-hd-kotlin-logo",
```

Comment:
We could also use the same link as other Jetbrains products: https://www.jetbrains.com/company/brand/logos/